skymp5-server/cpp/server_guest_lib/PieScript.cpp

```diff
@@ -1,6 +1,8 @@
 #include "PieScript.h"
+#include "FormDesc.h"
```

Comment:
If you want keep "PieScript.h" on top you should add empty line before next include statement. Otherwise reorder includes to satisfy clang format.